specs/SystemCursorId

```diff
@@ -0,0 +1,87 @@
+# Background
+
+In visual hosting, the current cursor image cannot be determined easily by the
+hosting application easily. For example, if the cursor over the WebView is also
+over a textbox inside of the WebView, the cursor should usually be IDC_IBEAM.
+And by default, the cursor is IDC_ARROW. Currently, we return the what the
+cursor should be as an HCURSOR object (After calling LoadCursor on those IDC
+values). The hosting application can then use the HCURSOR object to change the
+current cursor to the correct image.
+
+However, Office already has a well established way of using cursors internally
+by using IDC_* values and there is no easy way to convert the HCURSOR object we
+are returning into an IDC value after it's been created.
+
+This new API is to enable hosting applications like Office to get the current
+IDC value of the cursor instead of the HCURSOR as an option.
+
+
+# Description
+
+The `SystemCursorId` property will return the current system cursor ID reproted
+by the underlying rendering engine for WebView2. It is not meant to return a
+value for any custom cursors, such as those defined by CSS.
```

Comment:
You should note here that end developers can use the Cursor property instead to get HCURSOR and it will cover custom cursors